src/server.js

```diff
@@ -3,14 +3,13 @@
 import { renderToString } from 'react-dom/server';
 import Styletron from 'styletron-server';
 import { StyletronProvider } from 'styletron-react';
-import requireDir from 'require-dir';
-
 import doc from 'lib/__html';
 
-const pages = requireDir('./pages');
```

Comment:
maybe comment this? or do you think we'll need to swap it out for good?

Reply:
I'm working on an alternative using a dynamic `import()`. Give me a moment and I'll push my changes, just need to make sure I haven't broken the dev server!